macros/fivetran/base/fivetran_fb_ad_insights.sql

```diff
@@ -0,0 +1,64 @@
+{% macro fivetran_fb_ad_insights() %}
+
+    {{ adapter_macro('facebook_ads.fivetran_fb_ad_insights') }}
```

Comment:
if the code is exactly the same between default and snowflake, then you don't need to have a snowflake macro defined! please remove it for fb_ad_insights since the query seems to be identical